Gridy/Controller/PuzzleViewController.swift

```diff
@@ -55,6 +57,19 @@ class PuzzleViewController: UIViewController, UIGestureRecognizerDelegate {
         self.newGameButton.layer.cornerRadius = Constant.Layout.cornerRadius.buttonRadius
         self.newGameButton.clipsToBounds = true
     }
+
+    private func setupMovesNumberLabel() {
+        self.movesCountLabel.font = UIFont(
+            name: Constant.Font.Name.primary,
+            size: Constant.Font.Size.giantLabel)
+        self.movesCountLabel.textColor = UIColor (named: Constant.Color.primaryDark)
+        self.movesCountLabel.text = String(score)
+    }
+
+    private func updateMovesNumberLabel() {
```

Comment:
to rename `updateMovesCountLabel` for consistency?

Reply:
renamed